README.md

```diff
@@ -170,6 +170,17 @@ We are using npm.
     * First change `WebApplication.Program.cs` by removing the `UseKestrel()` statement
     * You will need to change the callback in your forge app to match the URL you deploy to.
 	
+### Project file zip encoding
```

Comment:
I think link to the [FDA troubleshooting page](https://forge.autodesk.com/en/docs/design-automation/v3/developers_guide/troubleshooting/) also can be useful here.

Reply:
Good point, i have added the link to the paragraph :)